src/olympia/constants/blocklist.py

```diff
@@ -1,5 +1,5 @@
 # How many guids should there be in the stashes before we make a new base.
-BASE_REPLACE_THRESHOLD = 5_000
+BASE_REPLACE_THRESHOLD = 6_000
```

Comment:
This change isn't connected to the issue.

Reply:
I think this parameter is related to the following test and an increment was needed, since system guid was incremented from 5 to 6:

https://travis-ci.org/github/mozilla/addons-server/jobs/707131541 --(src/olympia/blocklist/tests/test_cron.py::TestUploadToRemoteSettings::test_stash_because_previous_mlbf).

I will revert this change and re-verify via the TravisCI build.

Reply:
It's not related to the test.  You're changing it from 5000 to 6000 and we certainly don't test with that many guids (we mock the threshold down to a lower amount and test with that instead)

Reply:
Thank you for the clarification, I have made the update and re-requested a review